docs/user/intro.rst

```diff
@@ -1,263 +0,0 @@
-
-  - Meta: Sometimes to make the experiment easier to manage, it is useful to make an instrument to represent some element of the system that may be controlled in part by several separate instruments. For example, a device that uses one instrument to supply DC voltages, another to supply AC signals, and a third to measure it. We can make a new QCoDeS Instrument that references these lower-level Instruments, and we refer to this as a "Meta-Instrument". This imposes some requirements and limitations with remote instruments and multiple processes.
-
-An instrument can exist as local instrument or remote instrument. A local instrument is instantiated in the main process, and you interact with it directly. This is convenient for testing and debugging, but a local instrument cannot be used with a measurement loop in a separate process (ie a background loop). For that purpose you need a remote instrument. A remote instrument starts (or connects to) a server process, instantiates the instrument in that process, and returns a proxy object that mimicks the API of the instrument. This proxy holds no state or hardware connection, so it can be freely copied to other processes, in particular background loops and composite-instrument servers.
-
-.. responsibilities
-
-Instruments are responsible for:
-  - Holding connections to hardware, be it VISA, some other communication protocol, or a specific DLL or lower-level driver.
-  - Creating a parameter_, ref:`function`, or method for each piece of functionality we support. These objects may be used independent of the instrument, but they make use of the instrument's hardware connection in order to do their jobs.
-  - Describing their complete current state ("snapshot") when asked, as a JSON-compatible dictionary (everything that the custom JSON encoder class :class:`qcodes.utils.helpers.NumpyJSONEncoder` supports).
-
-.. state
-
-Instruments hold state of:
-  - The communication address, and in many cases the open communication channel.
-  - A list of references to parameters added to the instrument.
-
-.. failures
-
-Instruments can fail:
-  - When a VisaInstrument has been instantiated before, particularly with TCPIP, sometimes it will complain "VI_ERROR_RSRC_NFOUND: Insufficient location information or the requested device or resource is not present in the system" and not allow you to open the instrument until either the hardware has been power cycled or the network cable disconnected and reconnected. Are we using visa/pyvisa in a brittle way?
-  - If you try to use a background loop with a local instrument, because that would require copying the local instrument and there may only be one local copy of the instrument (if you make a remote instrument, the server instance is the one local copy).
-
-
-Parameter
```

Comment:
hmm, "parameters in qcodes" notebook refers to this "for more information" (see here https://qcodes.github.io/Qcodes/examples/Parameters/Parameters.html# ) , so perhaps we should move this "parmaeters" section into that notebook (and sanitize it a bit)? as opposed to just removing it?

hmm, and there's also some good info about parameters here https://qcodes.github.io/Qcodes/examples/writing_drivers/Creating-Instrument-Drivers.html#What%E2%80%99s-a-Parameter? , perhaps we can do some good referencing

Reply:
I agree we should move the text over into these sections. I also think we should try to consolidate the 'good parameter info' into a single place in order to make for more easily maintained documentation. E.g. the writing drivers guide should reference the parameter examples, not the other way around.

Reply:
> I also think we should try to consolidate the 'good parameter info' into a single place in order to make for more easily maintained documentation. E.g. the writing drivers guide should reference the parameter examples, not the other way around.

agree.

shall at least the "move" be done in this PR? and then consolidation can be done in a subsequent PRs?

Reply:
Yes, see below.